pkg/util/log/log.go

```diff
@@ -566,9 +570,13 @@ func Tracef(format string, params ...interface{}) {
 	logFormat(seelog.TraceLvl, func() { Tracef(format, params...) }, logger.tracef, format, params...)
 }
 
+func tracecWithDepth(message string, depth int, context ...interface{}) {
```

Comment:
Although I prefer this name, for consistency with the existing functions (example `TraceStackDepth` I think we should call this `tracecStackDepth`), WDYT?

(ditto for all the other similar functions for the other log levels)